### PR TITLE
DTSPO-12024 - Automate helm dependency updates

### PR DESCRIPTION
**Pull request by endakelly**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSPO-12024

### Change description ###
Update renovate.json to enable automated updates of helm dependencies

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: .github/renovate.json [hunk 1]
```diff
@@ -1,5 +1,9 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": ["config:base", ":dependencyDashboard"],
-  "labels": ["dependencies"]
+  "labels": ["dependencies"],
+  "commitMessageTopic": "helm chart {{depName}}",
```
Comment (timja) [inline .github/renovate.json:5]: why? this does more than helm?

File: .github/renovate.json [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
+  "helmv3": { 
+    "fileMatch": ["(^|/)Chart\\.yaml$"]
```
Comment (timja) [inline .github/renovate.json:7]: this should already be the default?

**Review by timja — CHANGES_REQUESTED** [review]

**Comment by timja** [comment]
Also see:
https://github.com/hmcts/sds-toffee-frontend/issues/61

image

**Comment by timja** [comment]
and https://github.com/hmcts/sds-toffee-frontend/pull/90

**Comment by endakelly** [comment]
Ah, didn't realise that already covered it. I was going to test this here and then roll out wider. I will get some info on our repos to make sure there aren't any with chart dependencies that are missing this - should solve my story.

**Pull request closed without merging**